### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by paritoshk** [kstathou/vector_engine#5]
I tried to utilize this method. May I ask why there is -e . at the end of the file? Docker shows an error for it when I tried it in my docker.

**Comment by Clement-Lelievre** [comment]
I also wondered that and ended up removing that line, without any issues.
Also, the pinned package versions are getting old and I needed to update them for the project to run smoothly

**Comment by ianni67** [comment]
Yes indeed, I had also to push the streamlit version up to the last available, and also add altair.vegalite.v4 to the requirements.

**Comment by michaelvincerra** [comment]
Agreed. In case it helps any future searchers, I updated all python modules in requirements.txt as:
```python
torch==2.0.1
transformers==4.31.0
sentence-transformers==2.2.2
pandas==2.0.3
faiss-cpu==1.7.4
numpy==1.25.1
folium==0.14.0
streamlit==1.25.0
-e .
```
